data/categories.txt

```diff
@@ -2440,7 +2440,7 @@ sw:^Makumbusho|kivutio|utalii
 historic-ruins|@tourism
 en:^Ruins|attraction|tourism|U+1F5FF
 ru:^Руины|развалины|достопримечательность|туризм
-ar:^آثار|أماكن الجذب|مناظر
+ar:^آثار|أماكن جذابة|مناظر|سياحة
```

Comment:
1. Is the word "ruins" here somewhere?
2. Again, please make sure the '^' sign points to the main translation here and in other places. (It still should be to the left of the required word.)